docs/src/pages/[platform]/getting-started/migration/migration.angular.mdx

```diff
@@ -1,21 +1,87 @@
 import { Tabs, TabItem } from '@aws-amplify/ui-react';
 import { Alert } from '@aws-amplify/ui-react';
 
+## 2.x to 3.x
+### Installation
+Install the 3.x version of the `@aws-amplify/ui-angular` library.
+
+<Tabs>
+<TabItem title="npm">
+
+```shell
+npm install aws-amplify @aws-amplify/ui-angular@3.x
+```
+
+</TabItem>
+<TabItem title="yarn">
+```shell
+yarn add aws-amplify @aws-amplify/ui-angular@3.x
+```
+
+</TabItem>
+</Tabs>
+
+### Update and usage
+
+The following breaking changes were introduced in `@aws-amplify/ui-angular` major version `3.0`:
+
+#### 1. Update the handleSignUp function override call
```

Comment:
It feels like these two steps should be under a heading:

```
#### Automatic signin on signup logic moved to `aws-amplify`. 

Make the following changes to upgrade to 3.0:

##### 1. Update the handleSignUp function override call
...
```

Reply:
👍